chainer/functions/loss/negative_sampling.py

```diff
@@ -373,8 +378,15 @@ def negative_sampling(x, t, W, sampler, sample_size, reduce='sum'):
 
     .. seealso:: :class:`~chainer.links.NegativeSampling`.
 
-    """
-    return (
-        NegativeSamplingFunction(sampler, sample_size, reduce)
-        .apply((x, t, W))
-    )[0]
+    """  # NOQA
```

Comment:
Add a documentation of the `return_samples` argument.

Reply:
Added documentation.